contracts/Treasury.sol

```diff
@@ -43,9 +45,12 @@ contract OlympusTreasury is OlympusAccessControlled, ITreasury {
         LIQUIDITYTOKEN,
         LIQUIDITYMANAGER,
         RESERVEDEBTOR,
+        RISKRESERVETOKEN,
         REWARDMANAGER,
-        SOHM,
-        OHMDEBTOR
+        SFLOOR,
+        FLOORDEBTOR,
+        XTOKEN, // Any token that requires a calculator to determine its value
```

Comment:
Updated to:

`// Any RESERVETOKEN that requires a calculator to determine its value`